tests/unit/API/Management/TicketsTest.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace Auth0\Tests\unit\API\Management;
+
+use Auth0\SDK\API\Helpers\InformationHeaders;
+use Auth0\SDK\Exception\EmptyOrInvalidParameterException;
+use Auth0\Tests\API\ApiTests;
+use GuzzleHttp\Psr7\Response;
+
+
+class TicketsTest extends ApiTests
```

Comment:
There were no tests (unit or integration) for the Tickets API. I've added unit tests to cover the methods changed. When/if changes are required to the other methods in `Tickets` we will have a test class in place to update.